From f9c2e8d14bb75a496cf61566464152edeb7db9fd Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Mon, 30 Sep 2013 19:06:51 +0800
Subject: [PATCH 255/304] ENGR00281863 BT:fix regression for AVRCP

In f6b5d7b6883c986f4efbcd3e28bd1ef8f26946a9, it delete all rwx
right for other which cause uinput not avaiable for BT, fix it
here.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 657d437..70f02d6 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -10,7 +10,7 @@
 /dev/video16              0660   system     graphics
 /dev/mxc_ipu              0660   media      system
 /dev/mxc_vpu              0660   media      system
-/dev/uinput               0660   system     input
+/dev/uinput               0660   system     bluetooth
 /dev/ttymxc2              0660   system     gps
 /dev/ttya0                0660   system     gps
 /dev/ptya0                0660   system     gps
-- 
1.8.0

